src/analysis/normed_space/basic.lean

```diff
@@ -1186,6 +1187,19 @@ normed_algebra.norm_algebra_map_eq _
 variables (𝕜 : Type*) [normed_field 𝕜]
 variables (𝕜' : Type*) [normed_ring 𝕜']
 
+@[continuity] lemma normed_algebra.algebra_map_continuous
```

Comment:
```suggestion
-- This could also be proved via `linear_map.continuous_of_bound`,
-- but this is further up the import tree in `normed_space.operator_norm`, so not yet available.
@[continuity] lemma normed_algebra.algebra_map_continuous
```